### DISCUSSION TITLE
It often takes so long to access the Security Onion Web Interface

### DISCUSSION THREAD

**Discussion by sanba06c** [Security-Onion-Solutions/securityonion#8827] [Unsupported Versions]
Hello,

The issue is when I logged in the Security Onion web interface, it was fine. However, when I clicked other tabs such as **Alerts**, the connection was so intermittent, sometimes it took very long, or connection timeout or sometimes it loaded normally.
 
Below are details of the local environment:

- Security Onion version 2.3.160
- This is an on-prem version with internet access
- It has been installed from a network installation, on Ubuntu 20.04.5 LTS
- There are four nodes
- Distributed architecture,1 manager node and 3 heavy nodes
-  There are no issues monitoring network traffic
- The so-status command shows all services are running
- The SOC Grid page shows no error
- No failures when running “sudo salt-call state.highstate”
- Log: A review of the logs in the path /opt/so/log/soc showed no unusual logs;

The said issue persisted even though I have re-installed it.

Any help or advice would be appreciated.

**Comment by isaacgolding** [comment]
There are a lot of variables there to look at.   I'd start with looking at the system load on each node and the connection speed on each node.   The next piece of the puzzle is the load and traffic volume on your switch(es).

**Reply by sanba06c** [reply]
Thank you for your useful advice! I'm looking into these probable causes.

**Comment by isaacgolding** [comment]
Take a look at your Graphna link .   You can get a lot of information out of the graphna dashboards that might help you find out which piece isn't playing nice.
